tool/benchmark_generator/README.md

```diff
@@ -0,0 +1,16 @@
+# Benchmark Generator
+
+Generates code that uses macros, for benchmarking.
+
+Example use:
+
+```
+dart bin/main.dart large macro 64
+dart ~/git/macros/pkgs/_macro_tool/bin/main.dart \
```

Comment:
It would be nice if this was something anybody could copy/paste and run from a specified path in the repo (in other words, if it was all relative paths)